R/license.R

```diff
@@ -11,7 +11,7 @@ renv_license_generate <- function() {
 
   contents <- c(
```

Comment:
Ooooh, now I see. You might want to switch to `use_mit_license()` and then delete this file.

Reply:
Maybe after this release?